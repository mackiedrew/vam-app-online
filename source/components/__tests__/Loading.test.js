import Subject from "../Loading";

describe("<Loading />", () => {
  it("renders without crashing", () => {
    const wrapper = shallow(<Subject />);
  });
});
