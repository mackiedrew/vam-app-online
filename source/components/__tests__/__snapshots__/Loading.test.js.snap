// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> renders correctly with props set 1`] = `
<div
  className="loading"
  id="loading"
>
  <div
    className="bar"
  />
  <div
    className="bar"
  />
  <div
    className="bar"
  />
  <div
    className="bar"
  />
  <div
    className="bar"
  />
  <div
    className="bar"
  />
  <div
    className="bar"
  />
  <div
    className="bar"
  />
  <div
    className="bar"
  />
  <div
    className="bar"
  />
</div>
`;

exports[`<Loading /> renders correctly without props 1`] = `
<div
  className="loading"
  id="loading"
>
  <div
    className="bar"
  />
  <div
    className="bar"
  />
  <div
    className="bar"
  />
  <div
    className="bar"
  />
  <div
    className="bar"
  />
  <div
    className="bar"
  />
  <div
    className="bar"
  />
  <div
    className="bar"
  />
  <div
    className="bar"
  />
  <div
    className="bar"
  />
  <div
    className="bar"
  />
  <div
    className="bar"
  />
  <div
    className="bar"
  />
  <div
    className="bar"
  />
  <div
    className="bar"
  />
  <div
    className="bar"
  />
  <div
    className="bar"
  />
  <div
    className="bar"
  />
  <div
    className="bar"
  />
  <div
    className="bar"
  />
</div>
`;
